content/about.md

```diff
@@ -18,110 +18,108 @@ Platform | Status | Development | Maintainer
-wrong and is why we decide to go asynchronous.
+ ST3 was unresponsive for a few seconds while the underlying `jedi` library
+was caching and processing the contents of the package. This was obviously
+the wrong approach and is why we decided to go asynchronous.
 
-Today, Anaconda is used by thousands of Python developers around the globe and
-this number continue growing daily (more than any other Python specialized
-plugin for Sublime Text).
+Today, Anaconda is used by thousands of Python developers around the globe, and
+this number grows daily — more than any other Python-specialized
+plugin for Sublime Text.
 
 # Anaconda Architecture
 
 Anaconda is an `asynchronous client-server` architecture application. That means
-that part of anaconda runs in the Sublime Text 3 embedded Python interpreter
-runtime and other runs in a decoupled standalone asynchronous server.
+that part of Anaconda runs in the Sublime Text 3 embedded Python interpreter
+runtime, and the other part runs in a decoupled, standalone, asynchronous server.
 
-All the heavy process is done in this standalone server so the embedded Python
-interpreter in Sublime Text 3 never get stuck, stale or unresponsive.
+All the heavy processing is done in this standalone server so that the embedded Python
+interpreter in Sublime Text 3 never gets stuck, stale or unresponsive.
 
-## How this works?
+## How does this work?
 
 Anaconda detects if any Sublime Text 3 window has a Python buffer view. If there
-is any Python buffer, it starts a new standalone anaconda's JsonServer (that
-knows how to speaks JSON with the part of anaconda that is running into the
-Sublime Text 3 runtime) for that window.
+are any Python buffers, it starts a new standalone Anaconda JsonServer for that window.  This server
+knows how to speak JSON with the part of Anaconda that is running in the
+Sublime Text 3 runtime.
 
 Anaconda's JsonServers use the Python interpreter that is available in the
-system `PATH` but this behavior can be configured globally or per project (
+system `PATH`, but this behavior can be configured globally or per project (
 take a look at [Configuring Anaconda the Right Way](http://localhost:1313/anaconda/anaconda_settings/)
 for detailed information about that).
 
 In this way, we solve two common problems in Sublime Text plugins:
 
 * Plugins have to work/use the Python version of the embedded Sublime Text runtime (Python 3.3.3).
-* Plugins usually make Sublime Text 3 unresponsive (or completely freeze) when they do weight process or calculations even using Threads as the embedded Sublime Text Python interpreter also have a [GIL](https://wiki.python.org/moin/GlobalInterpreterLock) that get locked
+* Plugins usually make Sublime Text 3 unresponsive (or completely frozen) when they do weight process or calculations, even when using Threads, as the embedded Sublime Text Python interpreter also has a [GIL](https://wiki.python.org/moin/GlobalInterpreterLock) that gets locked.
```

Comment:
Hmmm, I meant process that could take a while to complete as they process too much information or their implementations are not that optimal and it's growing complexity time is not so good.